frontend/src/metabase/reference/databases/FieldList.jsx

```diff
@@ -39,17 +37,23 @@ import { getIconForField } from "metabase/lib/schema_metadata";
 import * as metadataActions from "metabase/redux/metadata";
 import * as actions from 'metabase/reference/reference';
 
+
+const emptyStateData = {
+            message: `Fields in this table will appear here as they're added`,
+            icon: "fields"
+        }
```

Comment:
indentation pls – you could maybe run all data reference section files through `prettier` so that you don't have to care about formatting yourself (kyle is an expert on prettier stuff)